homeassistant/components/sabnzbd/config_flow.py

```diff
@@ -41,25 +45,28 @@ class SABnzbdConfigFlow(ConfigFlow, domain=DOMAIN):
 
     VERSION = 1
 
-    async def _async_validate_input(self, user_input):
-        """Validate the user input allows us to connect."""
-        errors = {}
-        sab_api = await get_client(self.hass, user_input)
-        if not sab_api:
-            errors["base"] = "cannot_connect"
-
-        return errors
+    async def async_step_reconfigure(
+        self, user_input: dict[str, Any] | None = None
+    ) -> ConfigFlowResult:
+        """Handle reconfiguration flow."""
+        return await self.async_step_user(user_input)
```

Comment:
Would it make sense to use "suggested_values" on show_form?

For example:
If there is user input, use that as suggested values
If not, and it's a reconfigure, use the previous entry data as suggested values